README.md

```diff
@@ -26,9 +26,11 @@ A Github Action that tests the deployment status of a Heroku Review App.
           env:
               GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           with:
-              interval: 10 # in seconds, optional, default is 10
-              accepted_responses: 200 # comma separated status codes, optional, default is 200
-              deployments_timeout: 120 # in seconds, optional, default is 120
+              checks: build, accessibility  # check the build status and if the app is accessible
```

Comment:
Why do we need to check the `build`? The Heroku-Github integration already marks the PR as bad if the build fails.

`accessibility` is not the best term, as it has a very specific meaning in software engineering: https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility

Maybe `resolves`?

Reply:
`build` was there before. Should I remove the check? In that case we won't need a new name.
If not another name suggestion might be `responds`. What do you suggest?

Reply:
Other name suggestions: responds_ok, http_ok, url_ok

Reply:
What do you mean it was there before? The `checks: build, accessibility` line is new?

Reply:
`responds` is good, but then we need something else instead of `build`. Maybe `builds` and `responds`?

Reply:
@suryasr007: what'd you think?

Reply:
> Maybe builds and responds?

I think it's good.

> What do you mean it was there before?

I mean it used to check both build status and response. I added the option to choose which checks to perform. If we select only response check, it will wait till the build finishes but won't fail the check if it fails. it will fail when the app doesn't respond properly.

I think it's better to rename `delay` to `delay_builds_check` and add `delay_responds_check`. Since different apps will have different build and load times.

Reply:
or `delay_check_builds` and `delay_check_responds`.

Reply:
@zupo We used `build check` to determine whether `application response`/`responds check` is required. We have checked the `application response`/ `responds check` only when the build is **succeded**.

But in the case of EBN with this [PR](https://github.com/niteoweb/ebn/pull/7274) merged, we are no longer building the application in Heroku. Hence, the build will fail (initial build). We only need to check the `application response`/ `responds check`.

I would prefer to have both the checks because in the default case there is no point to check `application response`/ `responds check` if the `build check` is failed.

As of namings, I would prefer `response` instead of `responds` as we are already using `accepted_responses` input.

@sayanarijit 
As this is a public repository, It would be better to add more examples to explain each check.

Reply:
I'll add some examples. By default, it will check both the Heroku app build status and deployment. I also think keeping the build status check is good since it provides more information about where the actual issue is in case of failue. @zupo are we fine with `response` and `build`?

Reply:
>  @zupo are we fine with response and build?

yep